src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -179,6 +188,11 @@ class ApplicationMenu extends Component<Props> {
           click: () => openProjectInEditor(selectedProject),
           accelerator: 'CmdOrCtrl+shift+E',
         },
+        {
+          label: isMac ? 'Open Settings' : 'Open settings',
+          click: () => showProjectSettings(),
+          accelerator: 'CmdOrCtrl+shift+S',
```

Comment:
the settings are usually `cmd + ,`, maybe it could be `CmdOrCtrl+shift+,` here?

Reply:
OK, I'll change it to `cmd + ,` same command as in VS code.

Reply:
Wait no, not `cmd + ,`. `cmd + ,` is the app settings. We might have some global settings like the default project path or some things like this later one.

That's why I proposed `cmd + shift + ,`

Reply:
You're right. I've added the shift. 

Do we have an issue for the application settings? If not can you please create one. So we can track your mentioned global settings. Home path should be easy to add because it's already used in `path.reducer`. For the rest we should create a new reducer and use these settings during project creation.